### PR TITLE
Add organization membership deactivate and reactivate API methods

### PR DESCRIPTION
**Pull request by mattgd**
## Description
Add organization membership deactivate and reactivate API methods.

## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[x] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

File: workos/user_management.py [hunk 1]
```diff
@@ -300,6 +307,7 @@ def list_organization_memberships(
         Kwargs:
             user_id (str): Filter Organization Memberships by user. (Optional)
             organization_id (str): Filter Organization Memberships by organization. (Optional)
+            statuses (list): Filter Organization Memberships by status. (Optional)
```
Comment (PavanKulkarni) [inline workos/user_management.py:310]: do we generally use this naming pattern for list params?

Comment (mattgd) [reply workos/user_management.py:310]: Do mean the pluralized parameter name? Seems like we do in the Python SDK:
https://github.com/workos/workos-python/blob/main/workos/organizations.py#L41
https://github.com/workos/workos-python/blob/main/workos/events.py#L38

Comment (PavanKulkarni) [reply workos/user_management.py:310]: no strong opinion then but I feel `statuses` seems a bit weird. Maybe `status_list`

Comment (mattgd) [reply workos/user_management.py:310]: Down to change it. That being said `statuses` is how it's defined in the API/other SDKs. Maybe it's better to be consistent, thoughts?

Comment (PaulAsjes) [reply workos/user_management.py:310]: I think we should be consistent and stick with `statuses`.

**Review by PaulAsjes — APPROVED** [review]

**Pull request merged**